src/libraries/System.Drawing.Common/tests/Imaging/CachedBitmapTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing.Imaging;
+using System.Linq;
+using System.Security.Permissions;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Drawing.Imaging.Tests
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "APIs not present in .NET Framework")]
```

Comment:
We don't need this attribute since this file won't even compile on .NET Framework.

Reply:
Yeah, don't know why I added this